core/src/main/java/org/infinispan/container/entries/ReadCommittedEntry.java

```diff
@@ -321,4 +327,31 @@ public boolean undelete(boolean doUndelete) {
       return false;
    }
 
+   public static class Externalizer extends AbstractExternalizer<ReadCommittedEntry> {
```

Comment:
If entry views were not serializable, function implementations could not return them :) Or do you already have functions that need this?